src/test/java/io/github/thebusybiscuit/slimefun4/core/services/sounds/TestSoundService.java

```diff
@@ -0,0 +1,34 @@
+package io.github.thebusybiscuit.slimefun4.core.services.sounds;
+
+import be.seeseemelk.mockbukkit.MockBukkit;
+import io.github.thebusybiscuit.slimefun4.implementation.Slimefun;
+import org.junit.jupiter.api.*;
+
+public class TestSoundService {
+
+    private static Slimefun plugin;
+
+    @BeforeAll
+    public static void load() {
+        MockBukkit.mock();
+        plugin = MockBukkit.load(Slimefun.class);
+    }
+
+    @AfterAll
+    public static void unload() {
+        MockBukkit.unmock();
+    }
+
+    @Test
+    @DisplayName("Testing each SoundConfiguration")
+    void testSounds() {
+        SoundService service = Slimefun.getSoundService();
+        SoundEffect[] effects = SoundEffect.values();
+
+        for (SoundEffect i : effects) {
+            SoundConfiguration element = service.getConfiguration(i);
+            Assertions.assertNotNull(element, "The sound effect must not be null!");
+        }
```

Comment:
Having a for loop in a unit test is not a good approach.
Please have a look at Parameterized Unit Tests:
https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests
Hint: If you are unable to map the parameters to the annotation parameter you may also want to look at the `@MethodSource` annotation